source/bdDetect.py

```diff
@@ -605,7 +612,7 @@ def _getDriverDict(self) -> DriverDictT:
 			ret = _driverDevices[self._driver] = DriverDictT(set)
 			return ret
 
-	def addUsbDevices(self, type: DeviceType, ids: Set[str]):
+	def addUsbDevices(self, type: DeviceType, ids: Set[str], useAsFallBack: bool = False):

```

Comment:
Just modernizing this with the change

```suggestion
	def addUsbDevices(self, type: DeviceType, ids: set[str], useAsFallBack: bool = False):
```